Documentation/kubernetes.md

```diff
@@ -36,6 +36,105 @@ kubectl create -f rook-operator.yaml
 # verify the rook-operator and rook-agents pods are in the `Running` state before proceeding
 kubectl -n rook-system get pod
 ```
+## RBAC for PodSecurityPolicies
+
+If you have activated the [PodSecurityPolicy Admission Controller](https://kubernetes.io/docs/admin/admission-controllers/#podsecuritypolicy) and thus are
+using [PodSecurityPolicies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/), you will require additional `(Cluster)RoleBindings` 
+for the different `ServiceAccounts` Rook uses to start the Rook Storage Pods.
+ 
+**Note**: You don not have to perform these steps if you do not have the `PodSecurityPolicy` Admission Controller activated!  
+
+##### PodSecurityPolicy 
+
+You will probably have a `PodSecurityPolicy` for elevated access in place, but you need one privileged Pod execution. Here is an example:
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: privileged
+spec:
+  fsGroup:
+    rule: RunAsAny
+  privileged: true
+  runAsUser:
+    rule: RunAsAny
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  volumes:
+  - '*'
+  allowedCapabilities:
+  - '*'
+  hostPID: true
+  hostIPC: true
+  hostNetwork: false
+```
+
+**Hint**: Allowing `hostNetwork` usage is required when using `hostNetwork: true` in the Cluster `CustomResourceDefinition`!
+You will then also require to allow the usage of `hostPorts` in the `PodSecurityPolicy`, i.e:
+ ```yaml
+ hostPorts:
+   - min: 1
+     max: 65536
+```
+
+##### ClusterRoleBinding
+
+Next up you require a `ClusterRoleBinding`, which enables the Rook Agent `ServiceAccount` to run the rook-agent `Pods` on all nodes
+with privileged rights. Create one according to this definition:
+
+```yaml
+# Allow the rook-agent serviceAccount to use the privileged PSP
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: rook-agent-psp
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
```

Comment:
Where do you create the `ClusterRole` and `Role` named `privileged-psp-user` or does the user need to create it but how?